scripts/pyroscope/install.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+# RUN THIS SCRIPT FROM THE ROOT OF THE REPOSITORY
+
+# shellcheck disable=SC1091
+source .env
+
+kubectl patch meshconfig osm-mesh-config -n "$K8S_NAMESPACE" -p '{"spec":{"observability":{"enableDebugServer":true}}}' --type=merge
+
+kubectl annotate -n "$K8S_NAMESPACE" svc osm-controller "pyroscope.io/scrape=true" "pyroscope.io/application-name=osm-controller" "pyroscope.io/spy-name=gospy" "pyroscope.io/profile-cpu-enabled=true" "pyroscope.io/profile-mem-enabled=true" "pyroscope.io/port=9092"
+
+helm install prof pyroscope-io/pyroscope -f scripts/pyroscope/values.yaml
```

Comment:
Looks like this is missing the step to add the pyroscope-io helm repo with `helm repo add`. But for things like this I wouldn't install frequently, I prefer to use the repo url directly so I don't have to do that and worry about running `helm repo update` to get newer versions later:
```suggestion
helm install prof pyroscope -f scripts/pyroscope/values.yaml --repo https://pyroscope-io.github.io/helm-chart
```